### PR TITLE
setActiveStatus logic big refactor

### PR DESCRIPTION
**Pull request by gcolotti**
Splitted setActiveStatus logic to small pieces to make it reusable.

### CODE REVIEW

File: src/services/UserService.js [hunk 1]
```diff
@@ -176,6 +179,60 @@ const deleteReservations = async user => {
   }
 };
 
+/**
+ * Creates the handle for the user
+ *
+ * @param {*} user
+ * @param {*} claimedPosition
+ */
+const getUserHandle = async (user, claimedPosition, maxQuePosition) => {
```
Comment (Benjamindodgson) [inline src/services/UserService.js:188]: I would rename this to "create" because thats more accurate.

File: src/services/ChatService.js [hunk 2]
```diff
@@ -206,6 +210,65 @@ const fetchChannel = async ChannelSid => {
...
+    };
+    await createMessage(newMessage, channel.sid);
+  }
+};
+
+/**
+ * Creates the initial channels for the new user
+ * @param {*} user
+ */
+const createUserChannels = async user => {
```
Comment (Benjamindodgson) [inline src/services/ChatService.js:230]: Lets rename this too. createInitialChannels 

I like naming 😬

Comment (Benjamindodgson) [reply src/services/ChatService.js:230]: or createNewUserChannels

Comment (Benjamindodgson) [reply src/services/ChatService.js:230]: My general naming convention is you should know exactly what the function does by its name.

**Review by Benjamindodgson — CHANGES_REQUESTED** [review]
A few minor name changes.

File: src/services/UserService.js (hunk 1 above)
Comment (gcolotti) [reply src/services/UserService.js]: Yeah, you are right. Done 😁

File: src/services/ChatService.js (hunk 2 above)
Comment (gcolotti) [reply src/services/ChatService.js]: Also done

Comment (gcolotti) [reply src/services/ChatService.js]: > My general naming convention is you should know exactly what the function does by its name.

You are right, names needed a clear and more accurate name

**Pull request merged**